### PR TITLE
Conversion enu/flu of worlds and controller

### PR DESCRIPTION
**Pull request by BenjaminHug8**
**Description**
This PR is to convert world and controller of `webots_ros2` to match flu/enu conversion R2022a.

### CODE REVIEW

**Comment by BenjaminHug8** [comment]
I noticed that some worlds that were deleted on `master` were not deleted in `develop`, so I cleaned it and deleted some worlds for `tiago` and `universal_robot`.

**Comment by BenjaminHug8** [comment]
@omichel is there anyway to tell the CI to use the branch `feature-enu-plan` ?

**Comment by omichel** [comment]
This is not easy as the CI is installing a debian package, see https://github.com/cyberbotics/webots_ros2/blob/master/scripts/ci_after_init.bash
So if you want the CI to use a build from the feature-enu-plan branch, you should create a deb package of this branch and use it.
This is probably not worth. We should simply test locally and ignore the CI tests for this PR (we may change the setting to allow us to merge it).

**Comment by BenjaminHug8** [comment]
> This is not easy as the CI is installing a debian package, see https://github.com/cyberbotics/webots_ros2/blob/master/scripts/ci_after_init.bash So if you want the CI to use a build from the feature-enu-plan branch, you should create a deb package of this branch and use it. This is probably not worth. We should simply test locally and ignore the CI tests for this PR (we may change the setting to allow us to merge it).

Ok I thought it would not be that easy to do it, thank you for the information.

**Review by joachimhgg — CHANGES_REQUESTED** [review]
Just small details. 
- clean the rotation of the worlds
- orientation of the door is wrong in turtlebot3
- universal_robot.wbt: box not FLU
- need to update doc ? https://github.com/cyberbotics/webots_ros2/wiki/Examples

**Comment by BenjaminHug8** [comment]
>     * orientation of the door is wrong in turtlebot3
I did it intentionally because otherwise the robot might (half) fall.

Screenshot from 2021-11-19 11-18-50

>     * need to update doc ? https://github.com/cyberbotics/webots_ros2/wiki/Examples

I have to check if FLU has an impact on the doc but the structure of the sub-packages of `webots_ros2` has changed so a refresh will be great in any case :+1: .

**Comment by BenjaminHug8** [comment]
Documentation has been updated.

**Review by joachimhgg — CHANGES_REQUESTED** [review]
Thanks! Last things to change, the walls of `rats_life_benchmark.wbt` are not FLU.

File: webots_ros2_epuck/worlds/epuck_world.wbt [hunk 1]
```diff
@@ -1,50 +1,49 @@
...
   ]
   title "ROS2 simulation of the e-puck robot"
 }
 Viewpoint {
-  orientation 0.6855367379017815 -0.23290009931686192 -0.6897803452731746 0.9152992115107464
-  position -1.4670162072354138 -2.3888456989525992 3.4003647484965547
-  follow "e-puck"
+  orientation -0.3259129066293485 0.6045847308938715 0.7268136490617071 1.2809214799066853
+  position -0.6554786770823011 -0.9985859086887117 1.3412109473782257
+  follow "e-puck-old"
```
[the remark this answers is not in the document]
Comment (BenjaminHug8) [reply webots_ros2_epuck/worlds/epuck_world.wbt:11]: As `followType` has never been defined, `follow` has no effect and the ViewPoint will never move.
Thus I will remove `follow "e-puck-old"`.

**Review by joachimhgg — APPROVED** [review]
Thanks!

**Comment by lukicdarkoo** [comment]
Before release:
- Fix CI for the develop branch (https://github.com/cyberbotics/webots_ros2/blob/4e7802349be45d5484475b00eea84a740a441756/scripts/ci_before_init_embed.bash#L3)
- Make sure all tests are passing on the develop branch (not critical, but good to have)
- We should merge this into the develop branch
- Merge `master` into `develop`

Tomorrow:
- Merge develop to master

File: scripts/ci_after_init.bash [hunk 2]
```diff
@@ -5,10 +5,10 @@ ROS_REPO=$2
...
 apt install -y wget dialog apt-utils psmisc
-wget https://github.com/cyberbotics/webots/releases/download/R${WEBOTS_VERSION}/webots_${WEBOTS_VERSION}_amd64.deb -O /tmp/webots.deb
+wget https://github.com/cyberbotics/webots/releases/download/nightly_15_12_2021/webots_${WEBOTS_VERSION}_amd64.deb -O /tmp/webots.deb
 apt install -y /tmp/webots.deb xvfb
 
 # The following packages are only available in the ROS 2 Foxy distribution. Therefore, we cannot include them in the package.xml, but we have to install them manually here.
 if [ "${ROS_DISTRO}" = "foxy" ]; then
     apt install -y ros-foxy-turtlebot3-cartographer ros-foxy-turtlebot3-navigation2
-fi
+fi
```
Comment (lukicdarkoo) [inline scripts/ci_after_init.bash:14]:
```suggestion
fi

```
We don't need this diff

**Review by lukicdarkoo — DISMISSED** [review]
Thank you

**Comment by BenjaminHug8** [comment]
I will just wait to merge the [gps PR](https://github.com/cyberbotics/webots_ros2/pull/354/) before merging this one into develop (the gps PR should be ready after CI tests).

**Review by lukicdarkoo — APPROVED** [review]
Thank you

**Pull request merged**